go.mod

```diff
@@ -1,11 +1,13 @@
-module github.com/manland/mattermost-plugin-gitlab
+module github.com/mattermost/mattermost-plugin-gitlab
 
 go 1.13
 
 require (
+	github.com/golang/mock v1.4.1
 	github.com/mattermost/mattermost-server/v5 v5.18.0
 	github.com/pkg/errors v0.8.1
 	github.com/stretchr/testify v1.4.0
+	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5 // indirect
```

Comment:
Is this line needed any longer?

Reply:
It should be removed, I'll take it out over the weekend.